data/Skin_Reaction/meta.yaml

```diff
@@ -0,0 +1,49 @@
+name: Skin_Reaction
+description: Repetitive exposure to a chemical agent can induce an immune reaction
+  in inherently susceptible individuals that leads to skin sensitization. The dataset
+  used in this study was retrieved from the ICCVAM (Interagency Coordinating Committee
+  on the Validation of Alternative Methods) report on the rLLNA.
+targets:
+- id: Skin_Reaction
+  description: whether it can cause skin reaction (1) or not (0).
+  units: Skin_Reaction
+  type: categorical
+  names:
+  - Skin Reaction
+  - skin sensitization
+  - agent Induced Skin Reaction
+  - fatal liver disease caused by drugs
```

Comment:
is this one correct here?

Reply:
It may be due to toxic metabolism, but here it was human error made by me. Sorry, it was from other version, I don't see it. please remove:
 - fatal liver disease caused by drugs